compiler/frontend/spark/src/main/java/org/apache/nemo/compiler/frontend/spark/transform/ReduceTransform.java

```diff
@@ -43,7 +42,6 @@
    */
   public ReduceTransform(final Function2<T, T, T> func) {
```

Comment:
Can you add a TODO here for this issue?
https://issues.apache.org/jira/browse/NEMO-432

Reply:
I added it.